storage/src/block.rs

```diff
@@ -1,35 +1,92 @@
 use {
     moved::{
-        block::{BlockRepository, ExtendedBlock},
+        block::{BlockQueries, BlockRepository, ExtendedBlock},
         primitives::B256,
+        types::state::BlockResponse,
     },
     rocksdb::{AsColumnFamilyRef, DB as RocksDb},
 };
 
 pub const BLOCK_COLUMN_FAMILY: &str = "block";
+pub const HEIGHT_COLUMN_FAMILY: &str = "height";
 
 #[derive(Debug)]
 pub struct RocksDbBlockRepository;
 
-impl RocksDbBlockRepository {
-    pub fn cf(db: &RocksDb) -> impl AsColumnFamilyRef + use<'_> {
-        db.cf_handle(BLOCK_COLUMN_FAMILY)
-            .expect("Column family should exist")
-    }
-}
-
 impl BlockRepository for RocksDbBlockRepository {
     type Storage = RocksDb;
 
-    fn add(&mut self, storage: &mut Self::Storage, block: ExtendedBlock) {
-        let cf = Self::cf(storage);
+    fn add(&mut self, db: &mut Self::Storage, block: ExtendedBlock) {
+        let cf = block_cf(db);
         let bytes = bcs::to_bytes(&block).unwrap();
-        storage.put_cf(&cf, block.hash, bytes).unwrap()
+        db.put_cf(&cf, block.hash, bytes).unwrap();
+        let cf = height_cf(db);
+        db.put_cf(&cf, block.block.header.number.to_le_bytes(), block.hash)
```

Comment:
Does it matter if this is `to_le_bytes` or `to_be_bytes`? Big endian is more human readable.